### ISSUE TITLE
Add labels in the example form

### ISSUE THREAD

**Issue by sukiletxe** [eduardoboucas/staticman#23]
Currently, the example form lacks labels in the fields. This makes it harder for screen readers to use the form.
Solution:
- Add a `id` to each field.
- Add a `<label>` element with the `for` attribute being the same as the `id` for each field The text of the element is the label.

This is not applicable to the submit button.

**Comment by eduardoboucas** [comment]
Thanks for this. I'll have a look when I can.

If you fancy submitting a PR, the docs are on the `gh-pages` branch: https://github.com/eduardoboucas/staticman/tree/gh-pages/docs
